swiftCtl.sh

```diff
@@ -0,0 +1,205 @@
+        The IP address of the swift instance to which files are pushed.
+        If not specified will default to >> $SWIFTIP <<.
+
+        -p <SWIFTPORT>
+        The port address of the swift instance to which files are pushed.
+        If not specified will default to >> $SWIFTPORT <<.
+        
+ DESCRIPTION
+ 
+        'swiftCtl.sh' is a simple shell script that interacts with a 
+        swift container.
+
+"
+
+function synopsis_show
+{
+        echo "USAGE:"
+        echo "$G_SYNOPSIS"
+}
+
+function CMD_eval
+{
+        CMD=$1
+        if (( Gb_verbose )) ; then
+                echo $CMD
+        fi
+        eval $CMD
+}
+
+while getopts E:C:P:p:A:VO:D: option ; do 
+	case "$option"
+	in
+                D)      PUSHDIR=$OPTARG                 ;;
+                V)      Gb_verbose=1                    ;;
+                O)      PULLDIR=$OPTARG                 ;;
+                A)      ACTION=$OPTARG                  ;;
+	        P)      SWIFTPATHPREFIX=$OPTARG         ;;
+	        p)      SWIFTORT=$OPTARG                ;;
+	        E)      EXT=$OPTARG                     ;;
+	        S)      SWIFTIP=$OPTARG                 ;;
+		\?)     synopsis_show 
+                        exit 0;;
+	esac
+done
+
+case $ACTION 
+in
+    "push")
+        here=$(pwd)
+        cd $PUSHDIR
+        for FILE in *${EXT} ; do
+                printf "Pushing file: $PUSHDIR/%s to ${SWIFTPATHPREFIX}/$FILE\n" $FILE
+                CMD="swift -U chris:chris1234 -A http://${SWIFTIP}:${SWIFTPORT}/auth/v1.0  -K testing upload users ${PUSHDIR}/$FILE --object-name ${SWIFTPATHPREFIX}/$FILE"
+                CMD_eval "$CMD"
+        done
+        cd $here
+        ;;
+    "list")
+        if (( ${#SWIFTPATHPREFIX} )) ; then
+            CMD="swift -U chris:chris1234 -A http://${SWIFTIP}:${SWIFTPORT}/auth/v1.0  -K testing list users --prefix $SWIFTPATHPREFIX"
+        else
+            CMD="swift -U chris:chris1234 -A http://${SWIFTIP}:${SWIFTPORT}/auth/v1.0  -K testing list users"
+        fi
+        CMD_eval "$CMD"
+        ;;
+    "pull")
+        # First, get a listing of files:
+        if (( ${#SWIFTPATHPREFIX} )) ; then
+            CMD="swift -U chris:chris1234 -A http://${SWIFTIP}:${SWIFTPORT}/auth/v1.0  -K testing list users --prefix $SWIFTPATHPREFIX"
+        else
+            CMD="swift -U chris:chris1234 -A http://${SWIFTIP}:${SWIFTPORT}/auth/v1.0  -K testing list users"
+        fi
+        LINELISTING=$(eval $CMD)
+        LIST=$(echo $LINELISTING | tr ' ' '\n')
+        
+        if (( ${#PULLDIR} )) ; then
+                OUTPUTDIR="--output-dir $PULLDIR"
+        else
+                OUTPUTDIR=""
+        fi
+
+        for FILE in $LIST ; do
+              printf "Pulling file: %s\n" $FILE
+              CMD="swift -U chris:chris1234 -A http://${SWIFTIP}:${SWIFTPORT}/auth/v1.0  -K testing download users $OUTPUTDIR --prefix $FILE"
+              CMD_eval "$CMD"
+        done
+        ;;
+esac
+
+# if (( Gb_verbose )) ; then
```

Comment:
Is there a reason to check in the commented out code?

Reply:
Good catch. No, there isn't